osquery/tables/system/darwin/sysctl_utils.cpp

```diff
@@ -41,6 +46,58 @@ const std::vector<std::string> kControlNames{
 const std::vector<std::string> kControlTypes{
     "", "node", "int", "string", "quad", "opaque", "struct"};
 
+void opaquePushback(QueryData& results,
+                      Row& r,
+                      std::string opaque_cv,
+                      const std::string& var) {
+      r["field_name"] = var;
+      r["current_value"] = opaque_cv;
+      results.push_back(r);
+  };
+
+void opaqueControlInfo(QueryData& results,
+                       Row& r,
+                       char* response,
+                       std::string& value) {
+  if (value.compare("S,clockinfo") == 0) {
+      struct clockinfo *ci = (struct clockinfo*) response;
```

Comment:
Use a names casts instead of C-style cast, please : [ref to CppCoreGuidelines](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#es49-if-you-must-use-a-cast-use-a-named-cast)